src/lib/isAlpha.js

```diff
@@ -1,26 +1,38 @@
 import assertString from './util/assertString';
 import { alpha } from './alpha';
 
-export default function isAlpha(_str, locale = 'en-US', options = {}) {
-  assertString(_str);
```

Comment:
In [upstream](https://github.com/validatorjs/validator.js/pull/2075) this was removed. Any idea why, @WikiRik ?

Reply:
I am referring to the function `isAfter`. Is it because `toDate` implicitely runs `assertString` in already?